docs/csharp/csharp-7.md

```diff
@@ -69,6 +69,10 @@ The most common use for this feature will be the `Try` pattern. In this
 pattern, a method returns a `bool` indicating success or failure and an
 `out` variable that provides the result if the method succeeds.
 
+> [!NOTE]
+> When using the `out` variable declaration, the scope of the declared variable is the same as it was in previous 
+> versions of C#. In other words, this is syntax sugar that enables inlining.  
```

Comment:
Ah, I see the concern. Previous versions of C# did in fact support the `out` variable, just not the inline declaration. But this does need clarification and example would be helpful. Thanks, I'll make an update.